src/app/pages/adoption-form/adoption-form.js

```diff
@@ -71,5 +71,4 @@ export default class AdoptionForm extends BaseView {
         this.regions.primaryBook.append(bookInfoView);
         this.toggler.update();
     }
-
```

Comment:
There's an empty line at the start and end of every other class, I believe.  For example, line 44 is blank at the opening of the class:
```
export default class AdoptionForm extends BaseView {

    @on('click .toggle-section')
```

I think you also want a blank line between the `}`s on line 73 and 74 (which did exist, but was removed here).